view.php

```diff
@@ -188,7 +188,7 @@
     if($zoom->enable_stream_url == 1 && $zoom->enable_download_url == 1){
         //$table->data[] = array(get_string('view_recording','zoom'));
         $display = '';
-        foreach ($records as $value) {
+        foreach ($records as $key => $value) {
```

Comment:
@nayanalinkstreet This $key is not being used anywhere there :/

Reply:
@Naveen1895 I believe this was old code and since I told her not to change anything in this file, she has raised this PR.

Reply:
@meyyappanv Yeah, I got that. But actually, since the $key is not used anywhere inside the loop, I thought it was unnecessary there. It is okay :)